app/controllers/api/v1/profiles_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    class ProfilesController < Api::V1::BaseController
+      MESSAGES = {
+        200 => nil,
+        404 => 'User not found',
+        403 => 'May be out of date'
+      }.freeze
+
+      def show
+        user_id = params[:id]
+        build_response(user_id)
+
+        render json: { message: @message, profile: nil } unless @profile
```

Comment:
Do you want to respond with 200 status even if there is no profile?
```suggestion
        render json: { message: @message, profile: nil }, status: SOMETHING unless @profile
```